src/reactviews/pages/TableExplorer/TableDataGrid.tsx

```diff
@@ -108,6 +112,9 @@ export const TableDataGrid = forwardRef<TableDataGridRef, TableDataGridProps>(
             if (onCellChangeCountChanged) {
                 onCellChangeCountChanged(0);
             }
+            if (onDeletionCountChanged) {
```

Comment:
Is there risk that this might not be set in the component props?  Could also simplify to `onDeletionCountChanged?(0)`, I _think_, since the check is more of a formality.